client/src/containers/SignupForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Button, Form, Grid, Header, Segment, Menu } from "semantic-ui-react";
+
+const SignupForm = () => {
+    return (
+        <Grid textAlign="center" verticalAlign="middle">
+          <Grid.Column style={{ maxWidth: 450 }}>
+            <Header as="h2" color="teal" textAlign="center">
+              Register your account
+            </Header>
+            <Form size="large">
+              <Segment stacked>
+                <Form.Input
+                  fluid
+                  icon="user"
+                  iconPosition="left"
+                  placeholder="Full name"
+                />
+                <Form.Input
+                  fluid
+                  icon="mail"
+                  iconPosition="left"
+                  placeholder="E-mail address"
+                />
+                <Form.Input
+                  fluid
+                  icon="lock"
+                  iconPosition="left"
+                  placeholder="Password"
+                  type="password"
+                />
+
+                <Button color="teal" fluid size="large">
```

Comment:
abstract these into separate components, so that you can use it everywhere. Same with input